GoogleForCreators/web-stories-wp
# Add captions option for video elements.

## Summary

This is a much simpler version of this PR than I have been working on before. 

This PR is very basic, that ability to upload one caption file per video. The ground work is in place to add multiples and to set kind to subtitle and set language. But for MVP. This would work. 

## Relevant Technical Choices

Use the clear button to get rid of the caption makes sense to me. We can work on the design in later versions. 

## To-do



## User-facing changes



## Testing Instructions



1. Open story editor. 
2. Upload test video file. 
3. Create video element. 
4. Select caption on design sidebar. 
5. Upload test VTT file. 
6. Publish story. 
7. See captions display. 

Also try removing caption file and re-adding. 
---



Fixes #4189
